source/patterns/@aws-solutions-constructs/aws-s3-stepfunctions/lib/index.ts

```diff
@@ -33,7 +33,7 @@ export interface S3ToStepfunctionsProps {
    */
   readonly existingBucketObj?: s3.IBucket,
   /**
-   * User provided props to override the default props for the S3 Bucket.
+   * Optional user provided props to override the default props for the S3 Bucket.
```

Comment:
These aren't optional (which you caught and fixed in the README)

Reply:
bucketProps are optional but stateMachineProps are required